feature-configs/deploy/gatekeeper/is_ready.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
We removed the is_ready.sh logic.
You need to contribute it to the validation suite